bin/next-build-tools.js

```diff
@@ -43,6 +43,7 @@ program
 	.option('--skip-enable-preboot', 'skip the preboot')
 	.option('--docker', 'deploy an app which uses docker')
 	.option('--gtg-urls <urls>', 'Comma separated list of urls to check before concluding the app is ok (these are in addition to __gtg)', list)
+	.option('--app <appname>', 'The app to deploy to (defaults to the name in package.json')
```

Comment:
I'm not quite sure that this is right:-

```js
program.command('test [app]')
        .option('--app <appname>')
        .action(function(app, opts) {
                console.log('arg1', app);
                console.log('opts.app', opts.app);
        });
```
```sh
mandrews@MATT ~/sandboxes/next-build-tools (more-docs) $ ./bin/next-build-tools.js test apppppp
arg1 apppppp
opts.app undefined
mandrews@MATT ~/sandboxes/next-build-tools (more-docs) $ ./bin/next-build-tools.js test apppppp --app apppppp
arg1 apppppp
opts.app apppppp
mandrews@MATT ~/sandboxes/next-build-tools (more-docs) $ ./bin/next-build-tools.js test  --app apppppp
arg1 undefined
opts.app apppppp
```

I actually prefer `--app` as an option as opposed to a [‘git-style sub command’](https://github.com/tj/commander.js#git-style-sub-commands)

I think to do that just need to make a tweak after the `.action(function(app, options) {` line to add:-

```js
if (app) {
  console.warn("`nbt deploy [app]` deprecated, please use `nbt deploy --app [app]`");
}
app = options.appname || app;
```